pkg/cluster/member.go

```diff
@@ -34,7 +34,7 @@ func (c *Cluster) updateMembers(known etcdutil.MemberSet) error {
 		if c.cluster.Spec.SelfHosted != nil {
 			name = m.Name
 			if len(name) == 0 || len(m.ClientURLs) == 0 {
-				c.logger.Errorf("member peerURL (%s): %v", m.PeerURLs[0], errUnexpectedUnreadyMember)
+				c.logger.Warningf("member peerURL (%s): %v", m.PeerURLs[0], errUnexpectedUnreadyMember)
```

Comment:
The reason we have this logging is to print out the PeerURL for debugging.
Externally, this could be either just a warning or an error.
Ideally we should have a critical error type, and make the information available to print outside. But here this doesn't need to be error.